### PR TITLE
verifytree: don't fail with versioned docdirs

### PR DESCRIPTION
**Pull request by dmnks**

### CODE REVIEW

File: verifytree.py [hunk 1]
```diff
@@ -47,8 +47,16 @@ def optionxform(self, optionstr):
...
-# RELAX NG schema for comps shipped with yum
-SCHEMA = '/usr/share/doc/yum/comps.rng'
+def get_schema_file():
+    """Return the path to the RELAX NG comps schema shipped with yum."""
+    base = '/usr/share/doc/yum'
+    # Our distro may use either versioned or unversioned docdirs (Fedora < 20
+    # does the former, see bug 998579), so try both variants here
+    for docs in [base, base + '-' + yum.__version__]:
+        path = docs + '/comps.rng'
+        if os.path.isfile(path):
```
Comment (megaumi) [inline verifytree.py]: Why not os.path.exists()?

File: verifytree.py [hunk 2]
```diff
@@ -47,8 +47,16 @@ def optionxform(self, optionstr):
...
+            return path
+    return None  # Shouldn't happen
```
Comment (megaumi) [inline verifytree.py]: How about raising an exception instead? That should be a sufficient indicator that this shouldn't happen. Also, we could pass the list (or better yet, a tuple) of paths we've checked with the exception object, to print a more helpful message for troubleshooting.

**Comment by dmnks** [comment]
Good points, thanks. Reworked version pushed.

**Comment by dmnks** [comment]
I pushed a fixup, please review and I'll autosquash it then.

**Comment by megaumi** [comment]
Looks good.

**Comment by dmnks** [comment]
Fix up squashed, you can merge now.

**Pull request merged**